### DISCUSSION TITLE
New Release

### DISCUSSION THREAD

**Discussion by VikashPR** [devicons/devicon#1066] [Q&A] [answered]
Hey when is the next release of devicons ?

**Comment by Thomas-Boi** [comment, marked as answer]
It's happening soon. We are fixing a bug with the build bot. Once that's done, the release will be out.

**Reply by amacado** [reply]
New [release](https://github.com/devicons/devicon/releases/tag/v2.15.0) is done. Locking this discussion for that reason! :)
